### PR TITLE
add header linter to avoid license header mistakes

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: operatorapi/auth/operator_test.go [hunk 1]
```diff
@@ -1,3 +1,19 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (dvaldivia) [inline operatorapi/auth/operator_test.go:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

File: operatorapi/error.go [hunk 2]
```diff
@@ -1,3 +1,19 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (dvaldivia) [inline operatorapi/error.go:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

File: pkg/apis/networking.gke.io/v1beta1/doc.go [hunk 3]
```diff
@@ -1,21 +0,0 @@
-/*
```
Comment (dvaldivia) [inline pkg/apis/networking.gke.io/v1beta1/doc.go:1]: some code will break without this

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: pkg/apis/networking.gke.io/v1beta1/doc.go (hunk 3 above)
Comment (harshavardhana) [reply pkg/apis/networking.gke.io/v1beta1/doc.go:1]: there is no code that depends on this AFAICS

File: operatorapi/error.go (hunk 2 above)
Comment (harshavardhana) [reply operatorapi/error.go:2]: is this a new file? since its old file kept the appropriate year.

File: operatorapi/auth/operator_test.go (hunk 1 above)
Comment (harshavardhana) [reply operatorapi/auth/operator_test.go:2]: is this a new file? since its old file kept the appropriate year.

File: operatorapi/error.go (hunk 2 above)
Comment (dvaldivia) [reply operatorapi/error.go:2]: aah, I thought we should be updating any file we touched

**Comment by harshavardhana** [comment]
PTAL @dvaldivia @bexsoft

**Comment by harshavardhana** [comment]
Is this supposed to fail?

```
 2/77 failed (3m 25s)
child_process.js:660
    throw err;
    ^

Error: Command failed: xvfb-run --server-args="-screen 0 1280x720x24" npx testcafe "chrome:headless" portal-ui/tests/permissions/ --skip-js-errors
    at checkExecSyncError (child_process.js:621:11)
    at execSync (child_process.js:657:15)
    at Object.<anonymous> (/home/runner/work/_actions/DevExpress/testcafe-action/latest/index.js:64:1)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11 {
  status: 2,
  signal: null,
  output: [ null, null, null ],
  pid: 2676,
  stdout: null,
  stderr: null
}```

**Comment by dvaldivia** [comment]
@harshavardhana we have a couple of UI tests that are failing, we can fix those

**Review by dvaldivia — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

**Pull request merged**